src/styles/_Nav.scss

```diff
@@ -0,0 +1,99 @@
+@use "variables" as v;
+@use "mixins" as m;
+
+.nav {
+  @include m.flex-column;
+  width: 100%;
+  align-items: center;
+}
+
+.nav__header {
+  background-color: rgba(123, 71, 52, 0.75);
```

Comment:
In looking at the app as a whole, I'm wondering if we should maybe change this to a cooler color. Perhaps the blue, or gray?
It's mostly gray and white on the other pages, and this looks brown.
Teeny tiny potatoes - not a blocking change for this branch.
Let me know what you think and I can address it on the finesse branch IF we want to change.

Reply:
Thanks for making this change, and for updating the footer. Looks great.